bin/oscapd-cli

```diff
@@ -332,13 +403,39 @@ def confirm(prompt):
     sys.stderr.write("Operation aborted.\n")
     return False
 
-
-
 def main():
     parser = argparse.ArgumentParser(
         description="OpenSCAP-Daemon command line interface."
     )
     subparsers = parser.add_subparsers()
+    task_accessor = Task_accessor()
```

Comment:
Ah, thanks.

What do you recommend with the design of class?
There are regular methods which have to work with instance variables. 
Static methods don't work with some instance and there is no reason to give them access to "self". Maybe I can move static methods outside of the class, but now it is on one place at the code.